images/env-debian/Dockerfile

```diff
@@ -20,7 +20,8 @@ RUN apt-get install -y \
     libtool \
     m4 \
     make \
-    pkg-config
+    pkg-config \
+    util-linux
```

Comment:
On debian and ubuntu, it's uuid-dev.

```
dpkg -L util-linux | grep uuid | wc -l
0
```

```
apt-file search /usr/include/uuid/uuid.h
uuid-dev: /usr/include/uuid/uuid.h
```

Reply:
Yes. You're right. I hadn't checked properly.